apps/vs-code-designer/src/package.json

```
@@ -767,6 +767,16 @@
             "type": "boolean",
             "description": "Show a warning when an Azure Functions .NET project was detected that has mismatched target frameworks.",
             "default": true
+          },
+          "azureLogicAppsStandard.showStartDesignTimeWarning": {
+            "type": "boolean",
+            "description": "Show a warning asking users if they would like to start design time api on project load.",
```

Why is asking customers to open the designer considered a warning?

```suggestion
            "description": "Show a message asking customers whether they want to open the workflow designer at project load time.",
```

---

True. Will rename the variables to messages as well